docs/api/topics/firefox_os_addon_submission.rst

```diff
@@ -88,12 +85,34 @@ Add-on Creation
 
     **Request**
 
-    :param upload: the id of the :ref:`validation result <addon_validation>`
+    :param validation_id: the id of the :ref:`validation result <addon_validation>`
         for your add-on.
     :type upload: string
 
     **Response**
 
-    An :ref:`add-on <addon-response-label>`.
+    The newly created :ref:`add-on <addon-response-label>`. Note that an add-on
+    version is created automatically for you.
+
+    :status: 201 successfully created.
+
+Add-on Version Creation
```

Comment:
We don't really talk in prose, about the versioning system (i.e. each addon has a version, each version is individually reviewed, etc.). We should do that at the top of this file so headings like this make sense.